postgresql/goose/README.md

```diff
@@ -0,0 +1,35 @@
+# Goose
+
+Goose is a package that provide some useful functions to interact with a Postgres database.
```

Comment:
Please always use "PostgreSQL" to refer to PostgreSQL. This is important that we stick with one way of naming this database type everywhere.

```suggestion
Goose is a package that provides some useful functions to interact with a PostgreSQL database.
```